package.json

```diff
@@ -93,5 +93,6 @@
   "workspaces": [
     "webviews/process-editor",
     "playwright"
-  ]
+  ],
+  "prettier": "@axonivy/prettier-config"
```

Comment:
how does this work?

Reply:
prettier has some default places where it looks for configs. 
As this repo has no special config defined, I decided to put it into the package.json itself. 
So prettier loads this config from here or better from @axonivy/prettier-config

If we want to have some special settings for this repo, we can easily reintroduce an prettierrc.json or whatever which extends this shared config.